[4.0] Remove nowrap

brianteeman opened the pull request:
This PR removes the class "nowrap" from the markup in the admin as it doesn't exist in the css and where it is needed it is already part of an existing class


NOTE: This requires #21938 and #21967 to be merged first

brianteeman committed:
[4.0] Table Column alignment (part2) [a11y]

com_fields
com_csp
com_installer

Continuation of #21933
consistency
wrap
com-users
com-templates
com-tags
com-plugins
com-newsfeeds
m3enus and modules
Update default.php

brianteeman committed:
[4.0] Remove non-existant nowrap class

zero-24 committed:
Merge branch '4.0-dev' into remove_wrap

zero-24 committed:
Merge branch '4.0-dev' into remove_wrap

Quy commented on administrator/components/com_templates/tmpl/styles/default.php:
```diff
@@ -39,24 +39,24 @@
 						<thead>
 							<tr>
-								<td style="width:1%" class="nowrap text-center">
+								<td style="width:1%" class="text-center">
```
Remove `class="text-center"`. Not necessary for a non-breaking space.

brianteeman replied:
beyond the scope of this PR but if you remove it you will see it is needed

Quy commented on administrator/components/com_tags/tmpl/tags/default.php:
```diff
@@ -71,49 +71,49 @@
 				<thead>
 					<tr>
-						<th scope="col" style="width:1%" class="nowrap d-none d-md-table-cell center">
+						<th scope="col" style="width:1%" class="d-none d-md-table-cell center">
```
`text-center`?

brianteeman replied:
beyond the scope of this pr

Quy commented on administrator/components/com_menus/tmpl/items/default.php:
```diff
@@ -58,43 +58,43 @@
 							<?php endif; ?>
 							<?php if (($this->state->get('filter.client_id') == 0) && (Multilanguage::isEnabled())) : ?>
-								<th scope="col" style="width:10%" class="nowrap d-none d-md-table-cell text-center">
+								<th scope="col" style="width:10%" class="d-none d-md-table-cell text-center">
```
Remove `text-center`?

brianteeman replied:
beyond the scope of this PR and iirc already addressed in another pr of mine

Quy commented on administrator/components/com_languages/tmpl/languages/default.php:
```diff
@@ -45,37 +45,37 @@
 									<?php echo HTMLHelper::_('searchtools.sort', '', 'a.ordering', $listDirn, $listOrder, null, 'asc', 'JGRID_HEADING_ORDERING', 'icon-menu-2'); ?>
 								</th>
-								<td style="width:1%"  class="nowrap text-center">
+								<td style="width:1%"  class="text-center">
```
Remove extra space before `class`

brianteeman replied:
done

Quy commented on administrator/components/com_finder/tmpl/index/default.php:
```diff
@@ -33,25 +33,25 @@
 								<?php echo JHtml::_('searchtools.sort', 'COM_FINDER_INDEX_HEADING_INDEX_DATE', 'l.indexdate', $listDirn, $listOrder); ?>
 							</th>
-							<th scope="col" style="width:15%" class="nowrap text-center d-none d-md-table-cell text-center">
+							<th scope="col" style="width:15%" class="text-center d-none d-md-table-cell text-center">
```
Remove duplicate `text-center`

brianteeman replied:
done

Quy commented on administrator/components/com_associations/tmpl/associations/modal.php:
```diff
@@ -60,30 +60,30 @@
 				<tr>
 					<?php if (!empty($this->typeSupports['state'])) : ?>
-						<th scope="col" style="width:1%" class="center nowrap">
+						<th scope="col" style="width:1%" class="center">
```
Change to `text-center`

brianteeman replied:
beyond the scope of this pr

infograf768 committed:
[4.0] Multilingual : Correcting info-block category and parent category urls (#21916)

* [4.0] Correcting info-block category and parent category urls

* Getting rid of category slugs thanks Hannes

* missing one cat language variable

* Correcting routing for archived

* Getting the active language when item is tagged to ALL

* Load current language when item is set to ALL

* factory not used

* better code for menurules thanks @csthomas

anuragteapot committed:
[4.0][GSoC 2018] Improve Override Management   (#21851)

* Load correct core files of override files (#2)

Start implements loadcorefile() in administrator/components/com_templates/Model/TemplateModel.php

* CS (#3) Coding Standards

* codingstandards

* codingstandards (#4)

* Test (#6)

Phase 2 (2 part) Mechanism to find correct core file and implementation.

* Remove Notice: Only available for html-folder

* Remove Warning if core file not found (#11)

Thanks.
So one part of the issue https://github.com/joomla-projects/gsoc18_override_management/issues/12 is done.

* Implement the diff view in template manager 

Implement the diff view in template manager

* coding standard (#17)

* fix diff (#18) Fix bug in path in case of administrator template override.

Fix bug in path in case of administrator template override.

* Notification after update and TEST (#16)

Find changed files of overridden files and show message.

* coding standard (#21)

* correction

* correction (#26)

* Correcthtmlpath (#27)

* correction

* change oldhtml to newhtml

* List of updated override files. (#30)

* addcss (#34)

* Final Product  (#39)

Core and Diff view
Updated override history list.
Quick icon notification plugin.
Override control plugin.

* save 3 lines :)

* New feature show status. (#47)

show status in com_template view templates

* link

* corrected namespace

* Button to Switch (#35)

* wip add Switcher

* wip style switcher

* wip style switch make inline and change on off text

* wip start with js

* wip js

* wip delete buttons and make js more robust

* wip save to storage

* wip delete old code

* wip

* wip lint

* wip css

* set default value for switcher

* wip make switcher blue

* wip

* wip

* build

* correct names

* create new functions

* fist test code

* use onchange

* undo installer.min.js

* add forgotten new line at the end of css file

* correct align

* correct compare.es6 - only deleted the toggle part

* correct compare.js - only deleted the toggle part

* wip

* reduce timeout

* wrap in funcitons

* wip

* add use strict to both js-files(compare and toggle)

* add the timeout value of 500 again, because 200 are not enought in my case

* use css class 'active' for toggle views

* add strict

* time out for editor

* wip

* improvments use newActive and switch

* correction

* width of switcher-spans

* correct align

* do not use global

* wip

* removed timeouts

* JTEXT to TEXT

* forgotton last line

* deleted duplicated comments

* css fix align

* use unnamed functions in es6

* Sql files for fix database (#50)

* sql files for database fix

* delete space

* Suggestion for displaying Dates in view updates files (#52)

Correct Dates and do not use date of file any more

* Store Date as UTC and show it in server time zone (#57)

* modified and created date are created and stored in UTC

* convert dates for displaying in model

* spar a loop

* normalize timezone in view

* use language constants for dateformat

* JToolbarHelper to ToolbarHelper

* CS

* namespace

* plural

* name

* clean

* text

* fx

* sin

* files

* s

* Suggestion for language strings (#60)

* language strings

* correct typo

* delete media folder plg_quickicon

* add folder plg_quickicon to build/media_src

* delete files in media folder

* Move media folder - System (#66)

* multi

* cs

* delete files in media folder for joomla toolbar (#67)

* Fix button switchers style. (#70)

* button

* CS

* changed uitab.addTab for updated files

* Bring back core.js changes. (#69)

* core.js

* const

* fix

* form

* core

* hound

* CS

* scopr

* grid

* alpha

* cs

* lang

* only override file

* lang

* override lang installer

* Cs

* sub

* Update list of core extensions (#71)

* Language changes (#76)

* update

* Update en-GB.com_templates.ini

* override JLIB_HTML_PUBLISH_ITEM

this is the hover text on the publish icon in the list of files

* Change icon (#74)

change the icon to use an outline for more consistency

* lang

* not core (#75)

* not core

* Update en-GB.plg_installer_override.ini

* namespace

* cs

* Updated files (#82)

* Update default_updated_files.php

* Update en-GB.com_templates.ini

* Update en-GB.com_templates.ini (#81)

* Update en-GB.plg_quickicon_overridecheck.ini (#80)

* Update en-GB.plg_quickicon_overridecheck.ini (#79)

* remove space (#78)

* Update en-GB.plg_quickicon_overridecheck.ini

* Update en-GB.plg_quickicon_overridecheck.sys.ini

* remove hardcoded id

* null get function

* state

* clean

* More changes "core" to "original" (#85)

* cs

* update

* plural

infograf768 committed:
[4.0] Typo in update sql (#22024)

brianteeman committed:
remove space

brianteeman committed:
Update default.php

bembelimen committed:
[4.0] More state => stage renaming (#21662)

* More state => stage renaming

* Stages => Stage

* Conditions => Condition

* Revert article table header

* Remove unused language strings in filter xml

* Fix wrong indenting

* Indent closing bracket

* Indent closing bracket

* Fix wrong indenting

* Add missing semicolon

* Renamed language strings to fit the standard

bembelimen committed:
[4.0] Add publish up/down information to articles list (#21942)

* Add publish up/down information to articles list

* Change ordering

* Moved default values to the layout (for override purposes)
Correct the margins/paddings of inactive icons
Fix error message when executing a transition

* Fix sniffer issues

* Align equal sign

* Fix sniffer issues

* Tabs => Spaces

bembelimen committed:
[4.0] Add default stages/transitions when creating a new workflow (#21963)

* Make the new stage from a new workflow translateable

* Create the default workflow stages/transitions when creating a new workflow

* Typo fix

brianteeman committed:
[4.0] Workflow Enable/Disable (#21836)

* [4.0] Workflow Enable/Disable

Instead of published/unpublished/trashed this pr changes the status for workflows/stages/transitions to enabled/disabled/trashed

This PR required fixing an obvious bug in the js select-colouur
Note that you will need to do a full npm install to test this

* locks.

* Revert "locks."

This reverts commit 8f170ea52d13702ae4d39804404466328b62c98f.

* locks

Hackwar committed:
Enabling plg_content_finder plugin by default (#20904)

bembelimen committed:
[4.0] Fix broken iframe + modal + backdrop combination (#21847)

* Fix broken iframe + modal + backdrop combination

* Update PopupButton.php

bembelimen committed:
[4.0] Implement the workflow into the blog sample data (#21953)

* Implement the workflow into the blog sample data

* Tabs => Spaces

* Typo fix

HLeithner committed:
[4.0][Workflow] Remove com_content defaults from workflow code (#21641)

* Add component specific workflow conditions.

* Archived articles should not be visible when not filtered (=> 3.x behavior)

* CS

* Fixed missing namespace.

* CS

* Doc fixes and removed archive state from featured published filter.

* Make drone happy

* Drone...

* cs

* Postgresql update, add extension parameter to getCondictions()

* Fix condition filter

* Remove com_content references in workflow

* cs

* Use quoteName

* Fix workflow edit link

* Throw an exception if no extension is set in workflow controller.

* Fixed type hint and leftover variable.

HRIT-Florian committed:
[4.0] Fix for #22053 COM_USERS enabling and adding YubiKey runs into Fatal error Joomla\CMS\Encrypt\Randvalinterface not found (#22054)

* fix for #22053

* fix for #22053
Namespecae was already in use, only wrong capitalization was the problem ;)

HLeithner committed:
Fix transitions controller missing ns. (#22062)

wilsonge committed:
Fix Checkin return type and typehint

wilsonge committed:
Fix error when deleting a stage

SamuelSchepp committed:
Changed column labels in com_workflow transitions (#22059)

ceus1984 commented:
And what are the test-instructions. I have taken part in several PBFs but then there were always exact test instructions given, what I am missing in several cases today

brianteeman commented:
closing for now

brianteeman committed:
new one

brianteeman commented:
going to have to redo this one to fix the conflicts

brianteeman commented:
see #22073
